h/static/scripts/controllers.coffee

```diff
@@ -113,18 +142,18 @@ class AppController
           delete $scope.selectedAnnotationsCount
 
     $scope.sort = name: 'Location'
-    $scope.threading = plugins.Threading
+    $scope.threading = threading
     $scope.threadRoot = $scope.threading?.root
 
 
 class AnnotationViewerController
   this.$inject = [
     '$location', '$routeParams', '$scope',
-    'annotator', 'streamer', 'store', 'streamfilter'
+    'streamer', 'store', 'streamfilter', 'annotationMapper'
   ]
   constructor: (
      $location,   $routeParams,   $scope,
-     annotator,   streamer,   store,   streamfilter
+     streamer,   store,   streamfilter,   annotationMapper
```

Comment:
Injecting both `store` and `annotationMapper` seems like an antipattern here. If it's a data mapper then it should abstract away the need to talk to the persistence layer (`store`).